odoo/tools/misc.py

```diff
@@ -1301,7 +1301,11 @@ def format_date(env, value, lang_code=False, date_format=False):
     if not date_format:
         date_format = posix_to_ldml(lang.date_format, locale=locale)
 
-    return babel.dates.format_date(value, format=date_format, locale=locale)
+    tzinfo = None
+    if env.context.get("tz"):
```

Comment:
Why not adding a tz=False additional parameter (suitable for stable policy as overrides / existing calls are still ok) to the method and calling it accordingly ?